### PR TITLE
Add gas to golang image

### PR DESCRIPTION
**Pull request by wongherlung**
We've decided to include `gas` into this base golang image so that all golang projects will include the necessary files and scripts to run the `gas` scan. This also makes changes easier to propagate to the various projects.

@soedar for your review.

### CODE REVIEW

File: golang/1.9/go-ast-scanner [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+mkdir -p /log
+gas -exclude=$GLOBAL_WHITELIST -fmt=junit-xml ./... | go run /gas-filter-report.go -whitelist whitelist.json > /log/report.xml
```
Comment (abeltay) [inline golang/1.9/go-ast-scanner]: What do you think about putting this into new repo, `go get -u <new repo>` which will compile the binary, then running the binary in the scanner?

File: golang/1.9/Dockerfile [hunk 2]
```diff
@@ -8,7 +8,10 @@ RUN apk --update add build-base git bash \
     && go get -v -u github.com/stretchr/testify \
                 github.com/tebeka/go2xunit \
                 github.com/t-yuki/gocover-cobertura \
-                gopkg.in/alecthomas/gometalinter.v2 \
-    && gometalinter.v2 --install
+                gopkg.in/alecthomas/gometalinter.v1 \
```
Comment (abeltay) [inline golang/1.9/Dockerfile]: Why is it changed to v1?

File: golang/1.9/go-ast-scanner (hunk 1 above)
Comment (wongherlung) [reply golang/1.9/go-ast-scanner]: ahh sounds like a good idea.

File: golang/1.9/Dockerfile (hunk 2 above)
Comment (wongherlung) [reply golang/1.9/Dockerfile]: whoops psps copy-pasta error. good catch.

**Comment by wongherlung** [comment]
@abeltay @soedar for your review again.

**Review by soedar — APPROVED** [review]
Lgtm

**Review by abeltay — APPROVED** [review]

**Pull request merged**